ui/admin/app/components/worker-filter/index.hbs

```diff
@@ -73,6 +77,63 @@
           }}</F.HelperText>
       </G.RadioField>
     </Hds::Form::Radio::Group>
+
+    {{#if (eq this.selectedGeneratorType this.generatorTagType)}}
+      <Hds::Form::Fieldset class='input-values' as |F|>
+        <F.Legend>{{t
+            'worker-filter-generator.filter-generator.input-values.title'
+          }}</F.Legend>
+        <F.HelperText>{{t
+            'worker-filter-generator.filter-generator.input-values.description'
+          }}</F.HelperText>
+        <F.Control>
+          <Hds::Form::TextInput::Field
+            @value={{this.tagKey}}
+            name='tag_key'
+            @width='320px'
```

Comment:
im wondering if we should leave this as pixels or use rem instead. Both work, just wondering what the convention is.

Reply:
I looked around and we primarily use `%` for the `@width` property. The issue with that is the input would get much smaller if the user wants change the width of the screen for split screen for example. I don't think `rem` would give us much of a change. I would lean towards leaving it as is.